mp/src/game/server/momentum/mom_player.cpp

```diff
@@ -41,6 +41,60 @@ CON_COMMAND_F(
     pPlayer->TogglePracticeMode();
 }
 
+CON_COMMAND(
+    mom_eyetele,
+    "Teleports the player to the solid that they are looking at.\n")
+{
+    CMomentumPlayer *pPlayer = CMomentumPlayer::GetLocalPlayer();
+    if (!pPlayer)
+        return;
+
+    if (pPlayer->m_bHasPracticeMode || !g_pMomentumTimer->IsRunning())
+    {
+        trace_t tr;
+        Vector pos = pPlayer->EyePosition();;
```

Comment:
Remove duplicate semicolon.
```suggestion
        Vector pos = pPlayer->EyePosition();
```